automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -98,6 +99,14 @@ def product_upgrade(product):
     If product is longrun then upgrade satellite, capsule and client
 
     :param string product: product name wanted to upgrade.
+    :param string sat_host: sat_host is satellite hostname.
+    :param string cap_hosts: cap_hosts is the capsule host's hostname.
+    :param string clients6: rhel 6 clients to be upgraded,
+        optional in case of docker clients.If not given,
```

Comment:
Change docstring style:
From:
```
    :param string clients6: rhel 6 clients to be upgraded,
        optional in case of docker clients.If not given,
        the clients will be generated on docker'
```
To:
```
    :param string clients6: rhel 6 clients to be upgraded,
        optional in case of docker clients.
        If not given, the clients will be generated on docker'
```